src/porepy/viz/diagnostics_mixin.py

```diff
@@ -90,8 +89,7 @@ def run_diagnostics(
         grouping: GridGroupingType
         | Literal["dense", "subdomains", "interfaces"]
         | None = None,
-        is_plotting_condition_number: bool = False,
-        is_plotting_max: bool = True,
+        default_handlers: tuple[Literal["cond", "max"], ...] = ("max",),
```

Comment:
Do you want to add a check that it does not contain any other entries?

Reply:
Also, please comment on the type choice. Are you sure a tuple is appropriate, even though it's length is not fixed?

Reply:
I agree, this is unnecessary here. I'll change it to Sequence type.